### ISSUE TITLE
Not sure I understand the Block Volume code

### ISSUE THREAD

**Issue by computamike** [jofftiquez/scope-of-works#13]
Hi - I am unsure what the 'footing' value in the block volume function is for?  I'm not a maths guy - but I did some resarch and the volume of a block or cuboid is width x height x depth.

The only thing I thought it might be could be to work out the volume of the inside of a box - taking into account the thickness of the matterial?  But I'm not sure that the formula performs that?

**Comment by jofftiquez** [comment]
@computamike you're right. The 'footing' parameter is not quite related to the volume computation itself. I will re-evaluate this one. The reason there was a 'footing' in here is that this function is supposed to be used for computing stuff done by civil engineers. I will clarify this matter to my engineer friend. Thanks for the input! Great help. I might update the README for the real purpose of this repo so people won't get confused if they see construction terminologies.

**Comment by computamike** [comment]
Would still be interested in this footing parameter - Any chance you could update this issue with a brief description of the footings parameter?  My only understanding (and I'm from the UK here - I'm not sure how terminology differs between countries) is related to building foundations.  

Is this formula documented somewhere because the function comment just says : 

```
/**
 * Calculate the volume of a rectangular box.
```

**Comment by jofftiquez** [comment]
@computamike right. I will give you an update on this one.

**Comment by computamike** [comment]
hi @jofftiquez  - has there been any update on this?
